app/views/inventories/_show.html.erb

```diff
@@ -0,0 +1,25 @@
+<table class="table table-striped">
+  <thead>
+    <tr>
+      <th>Food</th>
+      <th>Quantity</th>
+      <th>Action</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @inventories.each do |inventory| %>
```

Comment:
[OPTIONAL]
Hello,

First and foremost, great effort on building out the features for your Recipe App!

I've noticed an issue when trying to display the inventories on the `inventories/show.html.erb` view. The error being thrown is `undefined method 'name' for nil:NilClass`. This suggests that one or more of the `@inventories` items does not have an associated `food` object.

Here's what you might consider doing to resolve this:

1. **Safe Navigation**: 
   In the views, always ensure that you're handling potential nil values gracefully. You can use Ruby's safe navigation operator (`&.`) to avoid this error:
   ```erb
   <td><%= inventory.food&.name || 'N/A' %></td>
   ```

2. **Data Validation**: 
   Make sure that in your database and models, every `inventory` should have a valid association with a `food`. You can enforce this with model validations in Rails.

3. **Database Check**:
   It would be beneficial to review your database. Ensure that every `inventory` has a `food_id` that matches a record in the `foods` table. You might have orphaned `inventory` records.

4. **Debugging**:
   Consider adding some temporary debugging output to your loop to identify which `inventory` items are problematic:
   ```erb
   <% @inventories.each do |inventory| %>
     <tr>
       <% if inventory.food %>
         <td><%= inventory.food.name %></td>
       <% else %>
         <td>Missing food for inventory with ID <%= inventory.id %></td>
       <% end %>
       <!-- ... rest of your code ... -->
     </tr>
   <% end %>
   ```

Keep up the good work, and with a bit more debugging and data validation, your app will be more robust and error-free!

Best wishes,
Temitope